core/src/main/java/com/bobocode/bring/core/anotation/resolver/impl/ConfigurationBeanNameAnnotationResolver.java

```diff
@@ -0,0 +1,19 @@
+package com.bobocode.bring.core.anotation.resolver.impl;
+
+import com.bobocode.bring.core.anotation.Component;
+import com.bobocode.bring.core.anotation.Configuration;
+import com.bobocode.bring.core.anotation.resolver.AnnotationResolver;
+
```

Comment:
Need to provide some information in javadoc
Maybe something like this. What do you think ?
<img width="784" alt="Screenshot 2023-11-13 at 10 50 37" src="https://github.com/YevgenDemoTestOrganization/bring/assets/66901090/d15d8270-2e9a-4176-9c05-928cc0d74dcc">

Reply:
I want to add Javadocs everywhere a bit later, when everything is more stable, so Javadoc is not rewrited several times

Reply:
I want to add Javadocs everywhere a little bit later after more detailed testing, when everything is more stable, so Javadoc is not rewrited several times.
I will add some documentation in next PR with tests

Reply:
Okay, no problem )